win32ss/user/ntuser/timer.c

```diff
@@ -184,6 +184,7 @@ IntSetTimer( PWND Window,
 {
   PTIMER pTmr;
   UINT Ret = IDEvent;
```

Comment:
Why does the Ret variable is UINT, but it is used as the returned value of IntSetTimer which returns an `UINT_PTR`?
(and it is assigned initially to IDEvent which is an `UINT_PTR`...) Shouldn't this need to be fixed?

Note that this function corresponds to the win32 [`SetTimer()`](https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-settimer) which indeed uses `UINT_PTR` for these.

Reply:
Changed it to UINT_PTR.